src/core/pickBy.js

```diff
@@ -0,0 +1,15 @@
+import getAllKeys from './util/getAllKeys'
+import getIteratee from './util/getIteratee'
+import basePickBy from './base/basePickBy'
+
+export default function pickBy(data, predicate) {
+  if (data == null) {
+    return {}
+  }
+
+  const props = getAllKeys(data)
+  predicate = getIteratee(predicate)
+  return basePickBy(data, props, function(value, path) {
+    return predicate(value, path[0])
```

Comment:
This is a mistake on my part.  The lodash implementation maps the keys in an array.

```
    function pickBy(object, predicate) {
      if (object == null) {
        return {};
      }
      var props = arrayMap(getAllKeysIn(object), function(prop) { // <-- not sure why they are doing this.
        return [prop];
      });
      predicate = getIteratee(predicate);
      return basePickBy(object, props, function(value, path) {
        return predicate(value, path[0]);
      });
    }
```

So I think that's why they are using path[0] here.  Since I am not wrapping getAllKeys() in an array, it should be `predicate(value, path)` instead.